core/src/main/java/org/infinispan/partitionhandling/impl/PartitionHandlingManagerImpl.java

```diff
@@ -207,7 +211,13 @@ private void completeTransaction(final TransactionInfo transactionInfo, CacheTop
       remoteInvocation.whenComplete((responseMap, throwable) -> {
          final GlobalTransaction globalTransaction = transactionInfo.getGlobalTransaction();
          if (throwable != null) {
-            log.failedPartitionHandlingTxCompletion(globalTransaction, throwable);
+            if (throwable instanceof SuspectException) {
+               if (log.isTraceEnabled()) {
+                  log.tracef(throwable, "Node suspected while completing partial transaction %s", globalTransaction);
```

Comment:
@pruivo do we really need two different kinds of logs? Couldn't we always use `MapResponseCollector.ignoreLeavers()`?

Reply:
I don't think so. You don't want to ignore a node leaving in a distributed cache. You could end up in a minority partition and you need to keep the tx uncompleted.

Reply:
`MapResponseCollector.ignoreLeavers()` isn't really ignoring the leavers, it's just putting a `CacheNotFoundResponse` in the map instead of throwing a `SuspectException`. Then you can check the responses map for `CacheNotFoundResponse`, just like you already do a few lines below.

Reply:
you're right :) as usual eh eh

Reply:
> you're right :) as usual eh eh

If only I hadn't misread your code and thought that the tx completions were serialized in the very next comment :))